tests/libs/common/common_tests.h

```diff
@@ -6,8 +6,15 @@
 #pragma once
 #include <windows.h>
 
+#include "bpf.h"
 #include "ebpf_api.h"
 #include "ebpf_result.h"
+#include "libbpf.h"
+
+#pragma warning(disable : 26812) // Prefer enum class.
```

Comment:
which enum is this for?  Analyze.default.ruleset already says to treat this as a warning not an error.   If we want to disable it everywhere we can do it there, but we shouldn't have to do it in every file.